Sources/SwiftKuery/Table.swift

```diff
@@ -58,8 +58,10 @@ open class Table: Buildable {
 
     // MARK: Initializer
     /// Initialize an instance of Table.
-    public required init() {
+    /// - Parameter name: The name of the table (Optional).
+    public required init(name: String? = nil) {
         columns = [Column]()
+        _name = name ?? ""
```

Comment:
I think this can be removed.

Reply:
But, that initializes `_name`, right? (which is part of `self`, and needs to be done for mirroring). Or, do you mean you want to initialize `_name` just before verifying table properties?

Reply:
Yes, you can just do it at the end right before verifyTableProperties().

Reply:
Done.